packages/react/src/components/button/button.stories.tsx

```diff
@@ -92,3 +78,16 @@ WithIcon.argTypes = {
 WithIcon.args = {
   iconName: firstIconName,
 };
+
+export const AllCombinations = reactMatrix(
```

Comment:
Similar to Icon component, probably minor but also noticed that core stories render exact button children text what is being placed, however React only renders "Button":

<img width="240" alt="Screenshot 2021-02-12 at 11 09 42" src="https://user-images.githubusercontent.com/20243687/107761158-d1bbe600-6d22-11eb-9477-04484667fa32.png">

Reply:
That is interesting, I would call this a bug, I'll take a look.

Reply:
Weird and interesting one, fixed by cutting the `reactMatrix` features a little bit.
Namely the third argument is no longer a Function Component but simply a function that interpolates props.